scripts/ConnectBoxManage.sh

```diff
@@ -476,6 +476,13 @@ function set_ssid () {
     exit 1;
   fi
 
+  local ssid_length=`printf "%s" "$val" | wc -c`
+
+  if [[ $ssid_length -gt 32 ]]; then
+    echo "SSID must be 32 octets or less"
```

Comment:
This won’t appear to the user, will it? Were you planning to put something in the interface too, to cap the length of the input field (or something similar?)

Also, would you mind removing the unittest.skip decorators as a part of the PR, please?